crossbar/common/twisted/endpoint.py

```diff
@@ -293,7 +292,7 @@ def _create_tls_client_context(config, cbdir, log):
         for cert_fname in [os.path.abspath(os.path.join(cbdir, x)) for x in (config['ca_certificates'])]:
             cert = crypto.load_certificate(
                 crypto.FILETYPE_PEM,
-                six.u(open(cert_fname, 'r').read())
+                str(open(cert_fname, 'r').read())
```

Comment:
Not fully sure about this one though, help ?

Reply:
I am *fairly* certain that because you opened with `"r"` instead of `"rb"` you'll get unicode/str back in Python3 -- so I think you can just drop the `str(..)` entirely.

Reply:
I looked up the upstream documentation and indeed, this expects the buffer as `bytes` https://pyopenssl.org/en/stable/api/crypto.html#OpenSSL.crypto.load_certificate, So now I am opening with `rb` and **not** converting it to a string.

Reply:
:+1: